FALSEHOODS.md

```diff
@@ -0,0 +1,79 @@
+
+    **Tip:** Don’t store properties for a phone number such as validity or type. Check this information again from the library when you need it.
+
+1.  **A phone number uniquely identifies an individual**
+
+    It wasn't even that long ago that mobile phones didn't exist, and it was common for an entire household to share one fixed-line telephone number. In some parts of the world, this is still true, and relatives (or even friends) share a single phone number.
+
+1.  **An individual has only one phone number**
+
+    Obviously, this isn't necessarily true.
+
+1.  **Phone numbers cannot be re-used**
+
+    Old phone numbers are recycled and get reassigned to other people.
+
+1.  **Each country calling code corresponds to exactly one country**
+
+    The USA, Canada, and several Caribbean islands share the country calling code +1. Russia and Kazakhstan share +7. These are not the only examples!
+
+1.  **Each country has only one country calling code**
+
+    As of this present moment (in Mar. 2016), phones in the disputed territory and partially recognised state of Kosovo may be reached by dialing the country calling code for Serbia (+381), Slovenia (+386), or Monaco (+377), depending on where and when one obtained the number.
+
+    *Tip:* Use the phone widget to encourage users to enter their phone number in an international format such that we can understand it.
+
+1.  **A phone number is dialable from anywhere**
+
+    Some numbers can only be dialed within the country. Some can only be dialled from within a subset of countries, such as the international 00800 numbers. Some may be dialable only if the caller is a subscriber to a particular telecom company.
+
+1.  **There are only two ways to dial a phone number: domestically and from overseas**
+
+    Some numbers may need different prefixes depending on: the carrier you are using; what device you are dialling from/to; whether you are inside or outside a particular geographical region.
+
+    Examples:
+    *   In Brazil, to dial numbers internally but across a certain geographical boundary, a carrier code must be explicitly dialed to say which carrier you will use to pay for the call.
+    *   In Nepal, the leading zero in national format is omitted depending on whether the originating phone is mobile or fixed-line.
+    *   In New Zealand, you need to dial the area-code (e.g. 03) even if the number is within the same area-code region as you are, unless it is "close" (something approximating city/district boundaries), in which case it shouldn’t be dialled.
+
+    *Tip:* Use formatForMobileDialling to get the number a user should actually dial on their mobile phone.
+
+1.  **To make a number dialable, you only need to change the prefix**
```

Comment:
I don't understand "To make a number dialable" - presumably this is going from E.164 to national format? Isn't E.164 diallable inside the country too?